server/app/models/grid_service.rb

```diff
@@ -43,6 +43,8 @@ class GridService
   belongs_to :grid
   belongs_to :image
   belongs_to :stack
+  belongs_to :grid_domain_authorization
```

Comment:
Yes, something else is needed to support many authz per LB.

It's not currently used for anything. The idea was to use this in near future in the automated cert update, to update the SNI secret into correct LB service.